doc/build.md

```diff
@@ -119,4 +120,4 @@ troubleshooting, etc.
 
 Please note that all mining functions for Grin have moved into a separate, standalone package called
 [grin_miner](https://github.com/mimblewimble/grin-miner). Once your Grin code node is up and running,
-you can start mining by building and running grin-miner against your running Grin node.
\ No newline at end of file
+you can start mining by building and running grin-miner against your running Grin node.
```

Comment:
What's the change here?

Reply:
There seems to be none!  One moment while I fix this PR.

Reply:
Oops.  The change is there.  

https://github.com/mimblewimble/grin/pull/1975/files

I just added LLVM to the list of dependencies and to the dependency one-liner.

Reply:
No idea what the change there is.

Reply:
When I check here:  

https://github.com/mimblewimble/grin/pull/1975/files?short_path=10ebff8#diff-10ebff85ac4d31e01c65612e886ae89a

It only shows the llvm changes.